### DISCUSSION TITLE
Is it possible to create editable relational interface similar to this editable layout?

### DISCUSSION THREAD

**Discussion by ZeroLuckCZE** [formfcw/directus-extension-editable-layout#3] [Q&A] [answered]
Hello,

when I was searching for some way to make editing data inside Directus more Excel-like, I stumbled upon your editable layout extension and found it very useful. 
But then I thought it would be nice if I could use something similar to this extension instead of existing tabular relational interface (one-to-many, many-to-many) to edit relational items inline.

My question is: Is it possible to make an editable relational interface like this and if it is, have you ever considered so? Alternatively, how difficult do you think it would be to create such extension?

Thank you for your answer.

**Comment by formfcw** [comment, marked as answer]
It is possible. Feel free to contribute and create a pull request for it!
